src/hotspot/share/gc/shared/gc_globals.hpp

```diff
@@ -690,8 +690,12 @@
   product(uint, GCCardSizeInBytes, 512,                                     \
           "Card table entry size (in bytes) for card based collectors")     \
           range(128, NOT_LP64(512) LP64_ONLY(1024))                         \
-          constraint(GCCardSizeInBytesConstraintFunc,AtParse)
-  // end of GC_FLAGS
+          constraint(GCCardSizeInBytesConstraintFunc,AtParse)               \
+                                                                            \
+  develop(bool, UseAltGCForwarding, false,                                  \
```

Comment:
develop is constant value so we cannot verify this single commit. Shall we use a diagnostic/experimental?

Reply:
fixed